docs/zk2kraft.html

```diff
@@ -164,5 +164,16 @@ <h5 class="anchor-heading">Removal metrics</h5>
                 In Kraft mode, Zookeeper is not used, so the metrics is removed.
             </p>
         </li>
+        <li>
+            <p>
+                Remove the controller metrics. In Kraft mode, there is no broker which is elected as controller to communicate with ZooKeeper.
```

Comment:
This description is not accurate. We don't need to explain the all usage of those metrics, since that is not the purpose of this docs. Maybe we can add those metrics to the line#161? WDYT?

Reply:
Agree. I think we should also update description at line#158.